README.md

```diff
@@ -20,6 +20,7 @@ _by timeout_ and _by buffer size_.
 |1.3.*    |1.0.0                 |
 |1.9.0    |1.1.0                 |
 |1.9.0    |1.2.0                 |
+|1.9.0    |1.3.0                 |
```

Comment:
you could remove line 21-22 and leave just line #23

Reply:
yeap, thanks
done